### PR TITLE
Don't display projects that belong to deleted customers

### PR DESCRIPTION
**Pull request by fabidick22**
Ticket: https://github.com/orgs/ioet/projects/5#card-37540366

### CODE REVIEW

File: time_tracker_api/projects/projects_model.py [hunk 1]
```diff
@@ -34,19 +34,52 @@ class ProjectCosmosDBModel(CosmosDBModel):
...
+        """
+        Get all the projects an active client has
+        :param (dict) conditions: Conditions for querying the database
+        :param (dict) kwargs: Pass arguments
+        :return (list): ProjectCosmosDBModel object list
+        """
+        event_ctx = self.create_event_context("read-many")
+        customer_dao = customers_create_dao()
+        customers = customer_dao.get_all()
+        projects = self.repository.find_all(event_ctx, **kwargs)
```
Comment (enriquezrene) [inline time_tracker_api/projects/projects_model.py:71]: I think you should change this a little bit. 
1. Query all customers (DONE and OK)
2. Query projects `where customer_id IN ( customer_id from the above-query executed)`

With the IN constraint, you will have only projects that belong to customers that are not deleted.

Comment (fabidick22) [reply time_tracker_api/projects/projects_model.py:71]: So you say, do a query for each customer to find the projects?
I think it is a good solution, but we have to keep in mind the response time for each query, if we have many customers, this solution may be a little slow.

Comment (enriquezrene) [reply time_tracker_api/projects/projects_model.py:71]: Not at all, one query with an IN clause to get all the projects

Comment (fabidick22) [reply time_tracker_api/projects/projects_model.py]: Got it

**Pull request merged**